ansible/idr-downloads.yml

```diff
@@ -18,7 +18,7 @@
       comment: /OMERO directory from the IDR
     - name: sql
       path: /srv/omero-sql
-      comment: PostgreSQL 9.4 database dump of the IDR
+      comment: PostgreSQL 11 database dump of the IDR
```

Comment:
:+1: for catching this but perhaps harder to keep up to date than is necessary?

Reply:
True, I should be able to use https://github.com/IDR/deployment/blob/fdf2096e7ea4681d1b48c2587b5cf2bf67500d91/ansible/group_vars/omero-hosts.yml#L37 to reduce the maintenance

Reply:
:+1: